dao-contracts/tests/features/bid_escrow/external_to_va_worker.feature

```diff
@@ -16,6 +16,7 @@ Feature: External Worker who wants to become a VA submits job
     And following configuration
       | key                                    | value         |
       | TimeBetweenInformalAndFormalVoting     | 0             |
+      | VotingStartAfterJobSubmission          | 0             |
```

Comment:
I think it's better to set it to 0 in initial dao config rather then need every scenario to have it. But we can also do it later during the refactor stage.

Reply:
Yup, but the default value described in the doc specifies the value, so it shouldn't be 0 by default. The only alternative is to alter all the tests, and recalculate balances. I know it is a hassle to change all the tests set zero this value, but the variable itself is tested in isolation.